scripts/accept_eula.sh

```diff
@@ -10,7 +10,7 @@ RT_PASSWORD=$2
 IP_ADDR=$3
 
 echo "### Accept EULA for JCR distrubution on ${IP_ADDR}###"
-OUT=$(curl -XPOST -vu ${RT_USERNAME}:${RT_PASSWORD} http://${IP_ADDR}/artifactory/ui/jcr/eula/accept 2>&1)
+OUT=$(curl -XPOST -vu ${RT_USERNAME}:${RT_PASSWORD} ${IP_ADDR}/artifactory/ui/jcr/eula/accept 2>&1)
```

Comment:
Technically, it isn't the `IP_ADDRESS` - it's the URI that's represented here

also, a more secure way to run curl is to to use a `.netrc` file 
https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html

use `curl -n ...` for this feature